polkadot-parachain/src/command.rs

```diff
@@ -739,6 +797,32 @@ pub fn run() -> Result<()> {
 					.await
 					.map(|r| r.0)
 					.map_err(Into::into),
+					Runtime::BridgeHub(bridge_hub_runtime_type) => match bridge_hub_runtime_type {
+						chain_spec::bridge_hubs::BridgeHubRuntimeType::Kusama =>
+							crate::service::start_generic_aura_node::<
+								chain_spec::bridge_hubs::kusama::RuntimeApi,
+								AuraId,
+							>(config, polkadot_config, collator_options, id, hwbench)
+							.await
+							.map(|r| r.0),
+						chain_spec::bridge_hubs::BridgeHubRuntimeType::Rococo |
+						chain_spec::bridge_hubs::BridgeHubRuntimeType::RococoLocal =>
+							crate::service::start_generic_aura_node::<
+								chain_spec::bridge_hubs::rococo::RuntimeApi,
+								AuraId,
+							>(config, polkadot_config, collator_options, id, hwbench)
+							.await
+							.map(|r| r.0),
+						chain_spec::bridge_hubs::BridgeHubRuntimeType::Wococo |
+						chain_spec::bridge_hubs::BridgeHubRuntimeType::WococoLocal =>
+							crate::service::start_generic_aura_node::<
+								chain_spec::bridge_hubs::wococo::RuntimeApi,
+								AuraId,
+							>(config, polkadot_config, collator_options, id, hwbench)
+							.await
+							.map(|r| r.0),
+					}
+					.map_err(Into::into),
```

Comment:
this isn't possible is it? in the match above we know it has to be a `Runtime::Penpal(_) | Runtime::Default` so it can't be a bridgehub
```suggestion
```